function.tf

```diff
@@ -160,3 +160,61 @@ resource "google_cloud_scheduler_job" "this" {
     }
   }
 }
+
+resource "google_cloudfunctions_function" "rest_of_assets" {
+  count = var.enable_function ? 1 : 0
+
+  name                  = "${var.name}_rest_of_assets"
+  description           = "Testing this out"
```

Comment:
"testing this out" -- still intentional?

Reply:
No, thank you for catching